extension/sql/vectorize--0.13.0--0.14.0.sql

```diff
@@ -45,9 +45,8 @@ CREATE  FUNCTION vectorize."init_rag"(
 	"unique_record_id" TEXT, /* &str */
 	"column" TEXT, /* &str */
 	"schema" TEXT DEFAULT 'public', /* &str */
-  "index_dist_type" TEXT DEFAULT 'pgv_hsnw_cosine', /* vectorize::types::IndexDist */
```

Comment:
Do not change any of the existing migration files, such as `extension/sql/vectorize--0.13.0--0.14.0.sql`. These files tell Postgres how to update the extension across specific versions. The changes in this PR will be coming in the next release, so there will need to be a new migration created `extension/sql/vectorize--0.19.0--0.20.0.sql`. In that migration file we will need to `DROP FUNCTION` for any functions that have their signature changed, then add the new `CREATE FUNCTION` definition.

Reply:
Added new migration file